### PR TITLE
feat: internationalization Japanese

### LINKED ISSUE #3990: Language:  Japanese 🇯🇵
As a Japanese speaking person I want to be able to read ZITADEL login, user profile and console in Japanese

**Acceptance criteria**
- [ ] The whole login is shown in Japanese if I have Japanese as my default language set
- [ ] The user profile is shown in Japanese if I have Japanese as my default language set
- [ ] The console is shown in Japanese if I have Japanese as my default language set
- [ ] Emails are sent in Japanese if I have Japanese as my default language set

### PR DESCRIPTION
**Pull request by TakumiHaruta**
## Description

Close #3990 

- Add Japanese translations to all i18n files

## Checklist

- [x] I am happy with the code
- [x] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [x] No debug or dead code
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

**Comment by hifabienne** [comment]
Thanks @TakumiHaruta 
@zitadel/engineers can one of you have a look at this?

File: console/src/app/pages/org-create/org-create.component.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ export class OrgCreateComponent {
   public pwdForm?: UntypedFormGroup;
 
   public genders: Gender[] = [Gender.GENDER_FEMALE, Gender.GENDER_MALE, Gender.GENDER_UNSPECIFIED];
-  public languages: string[] = ['de', 'en', 'it', 'fr', 'pl', 'zh'];
+  public languages: string[] = ['de', 'en', 'fr', 'it', 'jp', 'pl', 'zh'];
```
Comment (peintnermax) [inline console/src/app/pages/org-create/org-create.component.ts:60]: Think this should be `ja` right?

File: console/src/app/app.module.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import localeDe from '@angular/common/locales/de';
 import localeEn from '@angular/common/locales/en';
 import localeFr from '@angular/common/locales/fr';
 import localeIt from '@angular/common/locales/it';
+import localeJp from '@angular/common/locales/ja';
```
Comment (peintnermax) [inline console/src/app/app.module.ts:7]:
```suggestion
import localeJa from '@angular/common/locales/ja';
```

File: console/src/app/app.module.ts [hunk 3]
```diff
@@ -61,16 +62,18 @@ import { ToastService } from './services/toast.service';
...
 i18nIsoCountries.registerLocale(require('i18n-iso-countries/langs/de.json'));
-registerLocaleData(localeZh);
-i18nIsoCountries.registerLocale(require('i18n-iso-countries/langs/zh.json'));
+registerLocaleData(localeEn);
+i18nIsoCountries.registerLocale(require('i18n-iso-countries/langs/en.json'));
 registerLocaleData(localeFr);
 i18nIsoCountries.registerLocale(require('i18n-iso-countries/langs/fr.json'));
 registerLocaleData(localeIt);
 i18nIsoCountries.registerLocale(require('i18n-iso-countries/langs/it.json'));
+registerLocaleData(localeJp);
```
Comment (peintnermax) [inline console/src/app/app.module.ts:71]:
```suggestion
registerLocaleData(localeJa);
```

File: console/src/assets/i18n/ja.json [hunk 4]
```diff
@@ -0,0 +1,2095 @@
...
+  "VERSION": "バージョン",
+  "TABLE": {
+    "NOROWS": "データなし"
+  },
+  "ERRORS": {
+    "REQUIRED": "一部の必須項目が不足しています。",
+    "TOKENINVALID": {
+      "TITLE": "トークンが期限切れになりました。",
+      "DESCRIPTION": "下のボタンをクリックして、もう一度ログインする。"
+    }
```
Comment (peintnermax) [inline console/src/assets/i18n/ja.json:247]:
```suggestion
    },
    "INVALID_FORMAT": "The formatting is invalid.",
    "NOTANEMAIL": "The given value is not an e-mail address.",
    "MINLENGTH": "Must be at least {{requiredLength}} characters long.",
    "UPPERCASEMISSING": "Must include an uppercase character.",
    "LOWERCASEMISSING": "Must include a lowercase character.",
    "SYMBOLERROR": "Must include a symbol or punctuation mark.",
    "NUMBERERROR": "Must include a digit.",
    "PWNOTEQUAL": "The passwords provided do not match.",
    "PHONE": "The phone number must start with 00 or +."
```

we had some new translations in the meantime.. could you set the correct values.. Thx

File: console/src/assets/i18n/ja.json [hunk 5]
```diff
@@ -0,0 +1,2095 @@
...
+      "NEW": "新しいパスワード",
+      "CONFIRM": "新しいパスワードの確認",
+      "NEWINITIAL": "パスワード",
+      "CONFIRMINITIAL": "パスワードの確認",
+      "RESET": "現在のパスワードをリセットする",
+      "SET": "新しいパスワードを設定する",
+      "RESENDNOTIFICATION": "パスワードリセットのリンクを送信する",
+      "REQUIRED": "一部の必須項目が不足しています。",
+      "MINLENGTHERROR": "最小で{{value}}文字の長さが必要です。",
+      "NOTEQUAL": "入力されたパスワードが一致しません。"
```
Comment (peintnermax) [inline console/src/assets/i18n/ja.json:529]:
```suggestion
      
```
got removed a in the meantime

File: console/src/assets/i18n/ja.json [hunk 6]
```diff
@@ -0,0 +1,2095 @@
...
+      "INVALIDPATTERN": "パスワードが、定義されたルールを満たしていません。",
+      "NOTANEMAIL": "指定された値はメールアドレスではありません",
+      "REQUIRED": "入力フィールドは空です",
+      "MINLENGTH": "パスワードは、最小で{{requiredLength}}文字の長さが必要です。",
+      "NOEMAIL": "ユーザー名をメールアドレスにすることはできません。",
+      "UPPERCASEMISSING": "大文字を含める必要があります。",
+      "LOWERCASEMISSING": "小文字を含める必要があります。",
+      "SYMBOLERROR": "シンボルまたは句読点を含める必要があります。",
+      "NUMBERERROR": "数字を含める必要があります。"
+    },
```
Comment (peintnermax) [inline console/src/assets/i18n/ja.json:593]:
```suggestion
```
got removed

File: console/src/assets/i18n/ja.json [hunk 7]
```diff
@@ -0,0 +1,2095 @@
...
+        "DESCRIPTION": "OAuthプロバイダーに必要なデータを入力してください。"
+      },
+      "JWT": {
+        "TITLE": "JWTプロバイダー",
+        "DESCRIPTION": "JWTプロバイダーに必要なデータを入力してください。"
+      },
+      "GOOGLE": {
+        "TITLE": "Googleプロバイダー",
+        "DESCRIPTION": "Google Identity Providerのクレデンシャルを入力してください。"
+      },
```
Comment (peintnermax) [inline console/src/assets/i18n/ja.json:1639]:
```suggestion
      },
      "GITLAB": {
        "TITLE": "Gitlab Provider",
        "DESCRIPTION": "Enter the credentials for your Gitlab Identity Provider"
      },
      "GITLABSELFHOSTED": {
        "TITLE": "Gitlab Self Hosted Provider",
        "DESCRIPTION": "Enter the credentials for your Gitlab Self Hosted Identity Provider"
      },
```

can you please provide the japanese translations? thx

File: console/src/assets/i18n/ja.json [hunk 8]
```diff
@@ -0,0 +1,2095 @@
...
+      "GITHUBES": {
+        "TITLE": "GitHub Enterprise Serverプロバイダー",
+        "DESCRIPTION": "GitHub Enterprise Server IDプロバイダーのクレデンシャルを入力してください。"
+      },
+      "GITHUB": {
+        "TITLE": "Githubプロバイダー",
+        "DESCRIPTION": "GitHub IDプロバイダーのクレデンシャルを入力してください。"
+      }
```
Comment (peintnermax) [inline console/src/assets/i18n/ja.json:1647]:
```suggestion
      },
      "AZUREAD": {
        "TITLE": "Microsoft Provider",
        "DESCRIPTION": "Enter the credentials for your Microsoft Identity Provider"
      }
```

File: console/src/assets/i18n/ja.json [hunk 9]
```diff
@@ -0,0 +1,2095 @@
...
+    },
+    "OWNERTYPES": {
+      "0": "不明",
+      "1": "インスタンス",
+      "2": "組織"
+    },
+    "STATES": {
+      "1": "アクティブ",
+      "2": "非アクティブ"
+    },
```
Comment (peintnermax) [inline console/src/assets/i18n/ja.json:1673]:
```suggestion
    },
    "AZUREADTENANTTYPES": {
      "0": "Common",
      "1": "Organizations",
      "2": "Customers"
    },
    "AZUREADTENANTTYPE": "Tenant Type",
    "AZUREADTENANTID": "Tenant ID",
    "EMAILVERIFIED": "Email verified",
```

this was also added in the meantime

**Review by peintnermax — CHANGES_REQUESTED** [review]
I've found some lines we changed in the meantime and a faulty locale code.. Rest looks good thx for your work 👍 :)

**Comment by TakumiHaruta** [comment]
@peintnermax 
Thanks for reviewing. I fixed typo and updated translations. Please have a look again!

**Review by peintnermax — APPROVED** [review]
Looks good to me 👍

**Comment by hifabienne** [comment]
@peintnermax can you please merge it when it is ok? external contributors are not able to do this.

**Comment by peintnermax** [comment]
Could you please update your branch to fullfill our requirements to merge your PR or enable this setting on your account to let us do that: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork 
thx for your help! 👍

**Comment by TakumiHaruta** [comment]
@peintnermax
I forked zitadel to github org account so there is no `Allow edits from maintainers` option.
Should I create a new PR from my private GitHub?

**Comment by peintnermax** [comment]
> 

This would be great so we can update the branch 👍

**Comment by TakumiHaruta** [comment]
@peintnermax 
I recreated PR in #5513
Please check it out!

**Comment by livio-a** [comment]
reviewe / basic test took me 15min

**Pull request closed without merging**